python/ray/serve/_private/http_proxy.py

```diff
@@ -243,6 +243,9 @@ def get_handle(name):
                 "status_code",
             ),
         )
+        self._ongoing_requests = 0
+        self._ongoing_requests_dummy_obj_ref = None
```

Comment:
[Nit] Is there a more expressive term for `dummy_obj_ref`? It's not immediately obvious what its purpose is from the name. Maybe `self.keep_alive_ref` or `self.keep_active_ref`?